Makefile

```diff
@@ -9,7 +9,7 @@ generate:
 	operator-sdk generate k8s
 
 build:
-	go build "./cmd/$(PROG)"
+	go build -ldflags "-X github.com/openshift/cluster-image-registry-operator/version.Version=$$(git describe --tags --always --dirty)" "./cmd/$(PROG)"
```

Comment:
i'm not sure if this will be viable when this is being built out of dist-git, but we can cross that bridge when we come to it.